code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -543,6 +556,173 @@ paths:
           $ref: '#/components/responses/500'
         '503':
           $ref: '#/components/responses/503'
+
+  /deployments:
+    post:
+      security:
+        - openId:
+            - edge-application-management:deployments:write
+      tags:
+        - Application
+      summary: Deployment of an Application
```

Comment:
```suggestion
      summary: Deploy an Application
```

Reply:
Changed from 'deployment' to make explicit this is an action, rather than information about a deployment.